<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenTypeDialog</class>
 <widget class="QDialog" name="GenTypeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Generator Selector</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sbMainResources.qrc">
    <normaloff>:/icons/sbicon.png</normaloff>:/icons/sbicon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.950000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: cyan;
}
QRadioButton:disabled {
	color: gray;
}
QRadioButton::indicator:disabled {
	border-image: url(:/radio/radio_disabled.png);
}
QRadioButton::indicator {
	width: 17px;
	height: 17px;
	border-image: url(:/radio/radio_off.png);
}
QRadioButton::indicator:hover {
	width: 17px;
	height: 17px;
	border-image: url(:/radio/radio_off_hover.png);
}
QRadioButton::indicator:pressed {
	width: 17px;
	height: 17px;
	border-image: url(:/radio/radio_pressed.png);
}
QRadioButton::indicator:checked {
	width: 17px;
	height: 17px;
	border-image: url(:/radio/radio_selected.png);
}
QRadioButton::indicator:checked:hover {
	width: 17px;
	height: 17px;
	border-image: url(:/radio/radio_selected_hover.png);
}
QPushButton {
	color: cyan;
	border-image: url(:/buttons/button75_idle.png);
}
QPushButton:disabled {
	color: gray;
	border-image: url(:/buttons/button75_disabled.png);
}
QPushButton:hover {
	border-image: url(:/buttons/button75_hover.png);
}
QPushButton:pressed {
	border-image: url(:/buttons/button75_pressed.png);
}</string>
  </property>
  <widget class="QRadioButton" name="disabledButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="toolTip">
    <string>Nothing!</string>
   </property>
   <property name="text">
    <string>Disabled</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="basicButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;A single wave built from the four basic waves.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Will be replaced in version 0.7.0 with a Subtractive generator.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Basic</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="modalButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="toolTip">
    <string>An impulse generator and a bank of resonating filters.
Coming in version 0.9.0.</string>
   </property>
   <property name="text">
    <string>Modal</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="sampleButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="toolTip">
    <string>Play from an external file.
This may in some cases be considered cheating.
Coming in version 0.4.0.</string>
   </property>
   <property name="text">
    <string>Sampler</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="additiveButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;A large bank of waves.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Coming in version 0.4.0.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Additive</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="waveguideButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="toolTip">
    <string>A banded waveguide synthesis system.
Coming in version 0.9.0.</string>
   </property>
   <property name="text">
    <string>Waveguide</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="backdrop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGraphicsView {
	background-image: url(:/popups/gentype_popup.png);
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>155</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>155</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <zorder>backdrop</zorder>
  <zorder>disabledButton</zorder>
  <zorder>basicButton</zorder>
  <zorder>modalButton</zorder>
  <zorder>sampleButton</zorder>
  <zorder>additiveButton</zorder>
  <zorder>waveguideButton</zorder>
  <zorder>cancelButton</zorder>
  <zorder>applyButton</zorder>
 </widget>
 <resources>
  <include location="sbMainResources.qrc"/>
 </resources>
 <connections/>
</ui>
